### ISSUE TITLE
how to save stats of multiple rooms all at once

### ISSUE THREAD

**Issue by ghost** [haxball/haxball-issues#1366]
overwrite is possible?

**Comment by ghost** [comment]
@basro where are you mate?

**Comment by thenorthstar** [comment]
You can try to use **localStorage**.

**Comment by misteranna** [comment]
> You can try to use **localStorage**.

hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?

**Comment by thenorthstar** [comment]
> > You can try to use **localStorage**.
> 
> hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?

@misteranna Yes, you can do it by using **localStorage**. But I suggest you to learn the fundamental stuff of localStorage (like get/set and remove the content) and JSON (parse and stringify) before starting to coding.

**Comment by misteranna** [comment]
> > > You can try to use **localStorage**.
> > 
> > 
> > hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?
> 
> @misteranna Yes, you can do it by using **localStorage**. But I suggest you to learn the fundamental stuff of localStorage (like get/set and remove the content) and JSON (parse and stringify) before starting to coding.

hi @thenorthstar, i don't think you understand my question.. i asking to send room 1 activity to room 2 automatic? 

so if you get ban on github1.com, ban will be valid on github2.com

**Comment by ghost** [comment]
@thenorthstar kesin bu sahis hesabi silenle ayni kisi 😄

**Comment by misteranna** [comment]
hi @basro

**Comment by thenorthstar** [comment]
> > > > You can try to use **localStorage**.
> > > 
> > > 
> > > hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?
> > 
> > 
> > @misteranna Yes, you can do it by using **localStorage**. But I suggest you to learn the fundamental stuff of localStorage (like get/set and remove the content) and JSON (parse and stringify) before starting to coding.
> 
> hi @thenorthstar, i don't think you understand my question.. i asking to send room 1 activity to room 2 automatic?
> 
> so if you get ban on github1.com, ban will be valid on github2.com

@misteranna Stop stupidizing me, I understand everything as it should be and have got the logic to understand what you said. Also I have already done it by using **localStorage** in my own room but don't share with you because of your bullshitting behaviors. And this is my last comment to you, please never ping me anymore.

@ikramimamoglu Evet, yazısından ve beni aptal yerine koymasından belli. Hem anlatılanı anlamıyor, hem karşısındakini aptal yerine koyuyor hem de her şeyi karşısındakine yaptırmaya çalışıyor. Daha da cevap vermeyin şuna.

**Comment by ghost** [comment]
> > > > > You can try to use **localStorage**.
> > > > 
> > > > 
> > > > hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?
> > > 
> > > 
> > > @misteranna Yes, you can do it by using **localStorage**. But I suggest you to learn the fundamental stuff of localStorage (like get/set and remove the content) and JSON (parse and stringify) before starting to coding.
> > 
> > 
> > hi @thenorthstar, i don't think you understand my question.. i asking to send room 1 activity to room 2 automatic?
> > so if you get ban on github1.com, ban will be valid on github2.com
> 
> @misteranna Stop stupidizing me, I understand everything as it should be and have got the logic to understand what you said. Also I have already done it by using **localStorage** in my own room but don't share with you because of your bullshitting behaviors. And this is my last comment to you, please never ping me anymore.
> 
> @ikramimamoglu Evet, yazısından ve beni aptal yerine koymasından belli. Hem anlatılanı anlamıyor, hem karşısındakini aptal yerine koyuyor hem de her şeyi karşısındakine yaptırmaya çalışıyor. Daha da cevap vermeyin şuna.

GitHub'a 7 saat once kayit olmus bu hesapla bir de utanmadan aciklama yazmis "👋 Hi, I’m @ misteranna" diye 🤣

**Comment by misteranna** [comment]
> > > > > You can try to use **localStorage**.
> > > > 
> > > > 
> > > > hi @thenorthstar, is it possible to bridge rooms?.. one backup for many rooms?
> > > 
> > > 
> > > @misteranna Yes, you can do it by using **localStorage**. But I suggest you to learn the fundamental stuff of localStorage (like get/set and remove the content) and JSON (parse and stringify) before starting to coding.
> > 
> > 
> > hi @thenorthstar, i don't think you understand my question.. i asking to send room 1 activity to room 2 automatic?
> > so if you get ban on github1.com, ban will be valid on github2.com
> 
> @misteranna Stop stupidizing me, I understand everything as it should be and have got the logic to understand what you said. Also I have already done it by using **localStorage** in my own room but don't share with you because of your bullshitting behaviors. And this is my last comment to you, please never ping me anymore.
> 
> @ikramimamoglu Evet, yazısından ve beni aptal yerine koymasından belli. Hem anlatılanı anlamıyor, hem karşısındakini aptal yerine koyuyor hem de her şeyi karşısındakine yaptırmaya çalışıyor. Daha da cevap vermeyin şuna.

hi @thenorthstar, so sorry.. i think i misunderstood but never insulted you.. i know you're just trying to help everyone.. i have tried for a long time but did not succeed and had to ask..
